torch/csrc/distributed/c10d/ProcessGroupNCCL.hpp

```diff
@@ -486,7 +486,18 @@ class TORCH_API ProcessGroupNCCL : public Backend {
     // Optional "parent" backend and color to create communicators from
     // via `ncclCommSplit`
     std::shared_ptr<ProcessGroupNCCL> split_from;
-    int64_t split_color{0};
+    // Color to use for `ncclCommSplit`, values:
+    // * Non-negative value: in group;
+    // * NCCL_SPLIT_NOCOLOR (-1): not in group;
+    // * NCCL_SPLIT_NOCOLOR - 1: uninitialized.
+    // [Note 1]: the type must be `int` instead of `int64_t` because NCCL API
+    // accepts int. Otherwise, an imlicit conversion may happen at the API call
```

Comment:
typo implicit

Reply:
Will fix in later PR.

Reply:
Addressed in https://github.com/pytorch/pytorch/pull/138374